server/lib/topStories.js

```diff
@@ -23,19 +25,81 @@ module.exports = function (startFrom, numberOfArticles, edition) {
 
 	let frontPageId = frontPageIdUK; // the default
 	if (edition) {
-    let editionUC = edition.toUpperCase();
-    if (editionUC === 'US' || editionUC === 'INTL') {
-      frontPageId = frontPageIdUS;
-    }
+		let editionUC = edition.toUpperCase();
+		if (editionUC === 'US' || editionUC === 'INTL') {
+			frontPageId = frontPageIdUS;
+		}
 	}
 
-	return fetch(`${apiOrigin}/${frontPageId}/main-content?apiKey=${apiKey}`)
+
+	let organisationPromise;
+	if (organisation) {
+
+		organisationPromise = co(function * () {
```

Comment:
Could we pull this out into a named function, it would help make the exported function easier to understand